DESCRIPTION

```diff
@@ -11,6 +11,7 @@ Encoding: UTF-8
 LazyData: true
 Suggests: 
     testthat,
+    stats,
```

Comment:
Why?

Reply:
I use it in one of the tests: https://github.com/reconhub/aweek/pull/21/files#diff-ba0c75efb3eba3bc6b8b3e20055a0846R64